### ISSUE TITLE
crop book cover images

### ISSUE THREAD

**Issue by caseyg** [caseyg/americanethnologist#73]

**Comment by caseyg** [comment]
<img width="704" alt="screen shot 2017-03-27 at 9 13 18 am" src="https://cloud.githubusercontent.com/assets/462020/24358011/a61aed24-12cd-11e7-8879-4ab62cb65bcf.png">

**Comment by daveleins8** [comment]
priority: low
8/28/17

**Comment by caseyg** [comment]
I'm worried that cropping book covers will clip important information that's placed along cover edges, sometimes. Maybe I can center the book covers within a square, so everything aligns?

**Comment by daveleins8** [comment]
I agree, cropping probably is too risky since the last thing we want is for
an author's name not to show up...they would not like that at all. Honestly
it looks pretty clean right now too. I can also just take the effort and
resize them myself to make the images fit properly. What is a max dimension
I should shoot for?
